estela_cli/context.py

```diff
@@ -30,12 +36,73 @@ def test_host(host):
     assert "projects" in response.json()
 
 
+def prompt_context(name, username=None, password=None, host=None):
+    try:
+        if host is None:
+            host = click.prompt("Host")
+        if username is None:
+            username = click.prompt("Username")
+        if password is None:
+            password = getpass()
+
+        estela_config_path = get_estela_config_path()
+        estela_config = get_estela_config()
+
+        if estela_config is None:
+            estela_config = {}
+
+        estela_config[name] = {
+            "host": host,
+            "username": username,
+            "password": password,
+        }
+        with open(estela_config_path, "w") as estela_config_yaml:
+            yaml.dump(estela_config, estela_config_yaml)
+            click.echo(
+                "Successful login. Context {} stored in ~/{}.".format(
+                    name, ESTELA_CONFIG_NAME
+                )
+            )
+        estela_client = EstelaClient(host, username, password)
+
+    except:
+        raise Exception("Unable to login with provided credentials.")
+
+    return estela_client
+
+
 @click.command(name="context", short_help=SHORT_HELP)
-def estela_command():
+@click.argument("name", required=False)
+def estela_command(name):
```

Comment:
Please add a docstring explaining what we can do with the command. You can see other commands such as `estela create job` as an example.